### PR TITLE
delete react template to get a clean repo

### PR DESCRIPTION
**Pull request by D3nnis249**
hey :) i´ve delete the react template and make all clean for work!

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
```
Comment (ThomasF85) [inline src/components/Header.js:1]: That was not part of the Exercise and should next time be put into a separate commit and organized under the respective issue. But good job getting it done so fast! #Meckernaufhohemniveau

**Comment by AlexF090** [comment]
Looks good❣️

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
```
Comment (ThomasF85) [inline src/App.js:1]: Good job also deleting unnecessary dependencies!

**Review by ThomasF85 — CHANGES_REQUESTED** [review]
I am missing a prettierrc file

**Review by ThomasF85 — APPROVED** [review]
